src/core/metrics/sources/nginx_plus.go

```diff
@@ -725,7 +738,7 @@ func (c *NginxPlus) cacheMetrics(stats, prevStats *plusclient.Stats) []*metrics.
 	return zoneMetrics
 }
 
-func (c *NginxPlus) slabMetrics(stats, prevStats *plusclient.Stats) []*metrics.StatsEntityWrapper {
```

Comment:
Can you remove the prevStats argument here? Looks like it isn't used in the function.

Reply:
Marking as resolved as this is already done as far as I can see. Let me know if anything else is required.